package.json

```diff
@@ -22,12 +22,12 @@
     "watch:prod": "shx rm -rf www/ && cross-env-shell SASS_PATH=node_modules \"stencil build --watch\"",
     "docs:build": "cross-env-shell NODE_ENV=prod SASS_PATH=node_modules \"stencil build --docs --config stencil.config.docs.ts\"",
     "docs:publish": "node publish-docs.js",
-    "lint": "npm run lint:src && npm run lint:scss",
-    "lint:fix": "npm run lint:src:fix && npm run lint:scss:fix",
-    "lint:prod": "npm run lint:src:prod && npm run lint:scss:prod",
-    "lint:src": "eslint '**/*.{ts,tsx,js}' --max-warnings=0 && prettier -l '{!(package*).json,.prettierrc}'",
-    "lint:src:fix": "eslint '**/*.{ts,tsx,js}' --fix --max-warnings=0 && prettier --write '{!(package*).json,.prettierrc}'",
-    "lint:src:prod": "eslint '**/*.{ts,tsx,js}' --rule 'linebreak-style: [error, unix]' --max-warnings=0 && prettier -l '{!(package*).json,.prettierrc}'",
+    "lint": "cross-env-shell \"npm run lint:src && npm run lint:scss\"",
+    "lint:fix": "cross-env-shell \"npm run lint:src:fix && npm run lint:scss:fix\"",
+    "lint:prod": "cross-env-shell \"npm run lint:src:prod && npm run lint:scss:prod\"",
+    "lint:src": "cross-env-shell \"eslint '**/*.{ts,tsx,js}' --max-warnings=0 && prettier -l '{!(package*).json,.prettierrc}'\"",
+    "lint:src:fix": "cross-env-shell \"eslint '**/*.{ts,tsx,js}' --fix --max-warnings=0 && prettier --write '{!(package*).json,.prettierrc}'\"",
```

Comment:
Hmm… there's got to be something wrong about this, because it doesn't seem to run on my machine.

![image](https://user-images.githubusercontent.com/154725/114713351-529e5a80-9d31-11eb-84d0-d6d4a9ba7b8c.png)

`npm run lint:scss:fix` seems to run just fine though… 🤔

I tried splitting the two calls up, like this

```suggestion
    "lint:src:fix": "cross-env-shell \"eslint '**/*.{ts,tsx,js}' --fix --max-warnings=0\" && cross-env-shell \"prettier --write '{!(package*).json,.prettierrc}'\"",
```

but that yielded the following error, which I guess was probably swallowed when running it in a single cross-env-shell:

![image](https://user-images.githubusercontent.com/154725/114714493-7615d500-9d32-11eb-915e-ce84d8b84d31.png)

Does it run on your machine?

Reply:
@adrianschmidt Good question, eslint doesn't seem to do anything for me either, but I don't get any errors...I'll take a look.

Reply:
I see you found a solution, nice! 👌

Reply:
I'm just going to double-check it by introducing an error in a temporary commit an checking that that the action on github catches it too 👍

Reply:
Awesome! Thanks for the feedback.

Reply:
Thanks for the PR! ❤️